arch/x86/core/intstub.S

```diff
@@ -314,7 +317,7 @@ alreadyOnIntStack:
 	popl	%eax
 
 	/* Pop of EFLAGS will re-enable interrupts and restore direction flag */
-	iret
+	IRET
```

Comment:
Might be a good idea to name IRET and IRET_USER to something that helps make them stand out. Otherwise, at least for IRET, it's easy to think it's just an ordinary iret. In fact, I was about to ask why this change was here until I saw the rest of the patches.

Suggestion: `KPTI_IRET` and `KPTI_IRET_USER`.  I think it makes sense even with these being just regular irets when KPTI is disabled: the important thing here is to be explicit that in the interrupt handling code, bare irets shouldn't be used due to the possibility of a kernel being built with KPTI.

Reply:
sure I'll rename them

Reply:
done.